CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+* The following diff formats now include information about copies and moves if
+  supported by the backend (the Git backend does): `--color-words`, `--summary`
```

Comment:
Nit: Drop the `if supported by the backend` part, as it will confuse users. As many don't understand the whole native/Git backend differences and _when_ they're used.

Reply:
Done